src/lib/components/Keyboard.js

```diff
@@ -62,6 +63,7 @@ class SimpleKeyboard {
     this.options.layoutName = this.options.layoutName || "default";
     this.options.theme = this.options.theme || "hg-theme-default";
     this.options.inputName = this.options.inputName || "default";
+    this.options.preventMouseDownDefault = this.options.preventMouseDownDefault || true;
```

Comment:
I think this will always yield "true". I will amend it to `this.options.preventMouseDownDefault === false ? false : true`